server.go

```diff
@@ -3589,14 +3564,16 @@ func HashPassword(password string) ([]byte, error) {
 type ContinuousQuery struct {
 	Query string `json:"query"`
 
-	mu              sync.Mutex
-	cq              *influxql.CreateContinuousQueryStatement
-	lastRun         time.Time
-	intoDB          string
-	intoRP          string
-	intoMeasurement string
+	mu      sync.Mutex
+	cq      *influxql.CreateContinuousQueryStatement
+	lastRun time.Time
 }
 
+func (cq *ContinuousQuery) intoDB() string          { return cq.cq.Source.Target.Measurement.Database }
```

Comment:
Yeah, `into` comes from the statement.  This was originally a variable which held data duplicated from a child struct.  I changed it to a function to avoid maintaining state in two places.